.gitignore

```diff
@@ -30,7 +30,7 @@
 # Ignore configuration files that may contain sensitive information.
 sites/*/settings*.php
 sites/*/services*.yml
-web/sites/default/settings.php
+web/sites/default/settings*.php
```

Comment:
Did you mean to add this in addition to the composer.lock file?

Reply:
I did, yes. But, thanks for asking as I realised I missed something else out.